utils/telegram.js

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs'
+import { Api, TelegramClient } from 'telegram'
+import { StringSession } from 'telegram/sessions/index.js'
+import random from 'random-bigint'
+import Config from '../config/index.js'
+import sendRequest from './request.js'
+
+const apiTelegram = 'https://api.telegram.org'
```

Comment:
prefer save to env

Reply:
yes, i was update to save env :smiley: 34ab37f